packages/shared/src/utils/constants/contract.tsx

```diff
@@ -446,8 +482,11 @@ export const getContractConfig = (is_high_low?: boolean) => ({
 // TODO we can combine getContractTypeDisplay and getContractTypePosition functions.
 the difference between these two functions is just the property they return. (name/position)
 */
-export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false) =>
-    getContractConfig(is_high_low)[type].name;
+
+export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false, show_button_name = false) => {
+    const contract_config = getContractConfig(is_high_low)[type] as TContractConfig;
+    return (show_button_name && contract_config.button_name) || contract_config.name || '';
+};
```

Comment:
```suggestion
export const getContractTypeDisplay = (
    type: keyof ReturnType<typeof getContractConfig>,
    is_high_low = false,
    show_button_name = false
) => {
    const contract_config = getContractConfig(is_high_low)[type];

    if (show_button_name && 'button_name' in contract_config) return contract_config.button_name;

    return contract_config.name;
};
```
@maryia-deriv Please avoid using `as` 🙏🏻 

Would you also please add `as const` to `getContractConfig`, `getSupportedContracts`, and `getUnsupportedContracts` methods 🙏🏻

Reply:
@farzin-deriv We've applied your suggestion, however, we need to keep the empty line as a fallback in case the type received is invalid/non-existent in the config.👍🏻 Also, even though the types of getContractConfig, getSupportedContracts, and getUnsupportedContracts methods are already inferred by TS successfully, we've added `as const` to the above methods as you requested. Regarding `TGetSupportedContracts`, we need the variable for `TGetSupportedContracts` instead of using `keyof ReturnType<typeof getContractConfig>` type directly in order to avoid duplication because it's used twice in this file.